src/NServiceBus.AcceptanceTests/Core/CoreTestSuiteConstraints.cs

```diff
@@ -6,8 +6,8 @@ public partial class TestSuiteConstraints
     {
         public bool SupportsDtc => false;
         public bool SupportsCrossQueueTransactions => true;
-        public bool SupportsNativePubSub => true;
-        public bool SupportsNativeDeferral => true;
+        public bool SupportsNativePubSub => false;
```

Comment:
Is this only for the spike? since we would loose the tests for the native pubsub? (I guess we agreed to tackel this by adding inmemory tests as a seprate csproj?

Reply:
yes, that's only for the spike. We would add a specific project to test inmem persistence and set this config only there.